.trivyignore

```diff
@@ -1 +1 @@
-CVE-2024-21538 exp:2024-12-31
\ No newline at end of file
+CVE-2024-21538
```

Comment:
to make trivy scanner pass again, this will be resolved when the underlying issue gets fixed in node 20 (or we end up updating to node 22)